lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -300,7 +300,7 @@
             // Add the social link fields
             socialFields = {
                 title: gettext('Social Media Links'),
-                subtitle: gettext('Optionally, link your personal accounts to the social media icons on your edX profile.'),  // eslint-disable-line max-len
+                subtitle: gettext('Optionally, link your personal accounts to the social media icons on your learning profile.'),  // eslint-disable-line max-len
```

Comment:
I don't think we should be inventing new terms; `learning profile` doesn't exist anywhere in the codebase.

It's typically referred as simply "your profile" or "your _edX_ profile" (when being more specific).

To date, I think our preference in un-harding the title `edX` has typically been "just remove the word `edx` and use the general term ("profile" not "edX profile"; "account" not "edX account"), given the general use of those words.

Reply:
fixed